test-butler-demo/src/androidTest/java/com/linkedin/android/testbutler/demo/SoftKeyboardDisablerTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2016 LinkedIn Corp.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.android.testbutler.demo;
+
+
+import android.support.test.rule.ActivityTestRule;
+import android.widget.EditText;
+import com.linkedin.android.testbutler.TestButler;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class SoftKeyboardDisablerTest {
+
+    @Rule
+    public ActivityTestRule<MainActivity> testRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void keyboardDoesNotShow() {
```

Comment:
Can you confirm that this only works on API 22+? If so, tag the tests with `@SdkSuppress`